modules/images/webp-uploads/load.php

```diff
@@ -573,3 +562,52 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
 	return webp_uploads_img_tag_update_mime_type( $html, 'post_thumbnail_html', $attachment_id );
 }
 add_filter( 'post_thumbnail_html', 'webp_uploads_update_featured_image', 10, 3 );
+
+/**
+ * Updates the image src data to use the webp format if an image has webp version.
+ *
+ * @since n.e.x.t
+ *
+ * @param array|bool   $image         Image data.
+ * @param int          $attachment_id Image attachment ID.
+ * @param string|array $size          Requested image size.
+ * @return array|bool Updated image data if the image has webp version, otherwise the original value.
+ */
+function webp_uploads_update_attachment_image_src( $image, $attachment_id, $size ) {
+	// Do nothing if image data is falsy or the size is not a string.
+	if ( empty( $image ) || ! is_string( $size ) ) {
+		return $image;
+	}
+
+	// Do nothing if not in the "the_content" context.
+	if ( ! doing_filter( 'the_content' ) ) {
+		return $image;
+	}
```

Comment:
I think we have to check this condition first in the function as it only works if the image is in the `the_content` context.

Reply:
Same here, we reverted this function.